CONTRIBUTING.md

```diff
@@ -112,6 +112,15 @@ e.g:
 ```bash
 dotnet test --environment="REDIS=172.17.0.1:6379"
 ```
+
+If you want to run your tests against an oss cluster, you can do it thus:
```

Comment:
`To run your tests against an oss cluster:`

Reply:
change also the example above?

> If you want to run your tests against a specific host and port, you can do it thus: